src/model/urls.ts

```diff
@@ -2,13 +2,15 @@ import { LanguageIdentifier } from '../resources/language';
 
 import { SupportType } from '.';
 
-export const TILE_URL = process.env.NODE_ENV === 'production'
-    ? "/tile/{z}/{x}/{y}"
-    : "http://localhost:8000/tile/{z}/{x}/{y}";
+export const TILE_URL =
+  process.env.NODE_ENV === 'production'
+    ? '/tile/{z}/{x}/{y}'
+    : 'http://localhost:8000/tile/{z}/{x}/{y}';
 
-const INVOICES_URL = process.env.NODE_ENV === 'production'
-    ? "/invoices"
-    : "http://localhost:8000/invoices";
+const INVOICES_URL =
+  process.env.NODE_ENV === 'production'
+    ? '/invoices'
+    : 'http://localhost:8000/invoices';
```

Comment:
Solche changes am besten nicht in so einen PR, lieber separat machen (wundert mich allerdings, warum das überhaupt ein Change ist lol, Prettier läuft ja eig. immer)

Reply:
Alternativ kannste den Commit ja auch noch droppen.

Reply:
Mach du mal. Ich würd squashen beim Merge, sodass ich dann da nix mehr droppen kann. Kannst den PR einfach force-pushen.